src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -744,6 +738,126 @@ def load_ldlc_medication_effect(key: str, location: str) -> pd.DataFrame:
     return df
 
 
+def load_bmi_exposure(key: str, location: str) -> pd.DataFrame:
+    entity = get_entity(key)
+    location_id = utility_data.get_location_id(location)
+
+    data = get_draws(
+        gbd_id_type="modelable_entity_id",
+        gbd_id=data_values.BMI_MEAN_ME_ID,
+        source=SOURCES.EPI,
+        location_id=location_id,
+        sex_id=SEX.MALE + SEX.FEMALE,
+        gbd_round_id=GBD_2020_ROUND_ID,
+        decomp_step="usa_re",
```

Comment:
This has all been clarified in the docs?

Reply:
Yup, confirmed with Syl directly as well.